data-science-onramp/data-ingestion/setup-test.py

```diff
@@ -0,0 +1,210 @@
+import os
+import re
+
+import uuid
+
+from google.api_core.exceptions import GoogleAPICallError
+
+from google.cloud import dataproc_v1 as dataproc
+from google.cloud import storage
+from google.cloud.exceptions import NotFound
+
+import pytest
+
+waiting_cluster_callback = False
+
+# Set global variables
+project = os.environ['GCLOUD_PROJECT']
+region = "us-central1"
```

Comment:
Should we consider making the region + zone environment variables as well? Or include a TODO for users to update these to reflect their project? Not all folks are US based. 

@bradmiro I know some products just straight up aren't available in other regions/zones, is Dataproc one?

Reply:
I've personally always hard-coded regions into tests as these don't typically make their way into tutorials. For the cluster to be used in the tutorial, I think a set of steps showing how to create a Dataproc cluster in a preferred region is appropriate.

Reply:
I believe Dataproc is available in any region that Compute instances are available.